girder/utility/install.py

```diff
@@ -315,16 +320,26 @@ def main():
 
     web.add_argument('--npm', default='npm',
                      help='specify the full path to the npm executable.')
-    web.add_argument('--all-plugins', action='store_true',
-                     help='build all available plugins rather than just enabled ones')
-    web.add_argument('--plugins', default='', help='comma-separated list of plugins to build')
+
     web.add_argument('--watch', action='store_true',
                      help='watch for changes and rebuild girder core library in dev mode')
+
     web.add_argument('--watch-plugin', default='',
                      help='watch for changes and rebuild a specific plugin in dev mode')
 
-    web.add_argument('--plugin-prefix', default='plugin',
-                     help='prefix of the generated plugin bundle')
+    pluginGroup = web.add_mutually_exclusive_group()
+
+    pluginGroup.add_argument('--all-plugins', action='store_true',
+                             help='build all available plugins rather than just enabled ones')
+
+    pluginGroup.add_argument('--plugins', default=None,
+                             help='comma-separated list of plugins to build')
+
+    pluginGroup.add_argument('--plugin-prefix', default='plugin',
+                             help='prefix of the generated plugin bundle')
```

Comment:
I'm not entirely sure what the `--plugin-prefix` is doing. Could you conceivably want to run `girder-install web --plugin-prefix --no-plugins` or `girder-install web --plugin-prefix --all-plugins`?